src/functionalTest/resources/application-functional.yaml

```diff
@@ -26,7 +26,7 @@ api-schema-file:
     submitanswer-schema: JsonSchema/submitAnswersSchema.json
 api-version-number: 1
 api-config-params:
-  number_of_days_limit: 180
+  number_of_days_limit: 210
```

Comment:
If it wasn't here, it would most likely be taken from application.yaml and wouldn't be needed to change everytime.